azure-communication-ui/demo-app/src/calling/java/com/azure/android/communication/ui/callingcompositedemoapp/CallLauncherApplication.kt

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.callingcompositedemoapp
+
+import android.app.Application
+import com.azure.android.communication.ui.callingcompositedemoapp.telecom.TelecomConnectionService
+
+class CallLauncherApplication : Application() {
+    var telecomConnectionServiceListener: TelecomConnectionService? = null
```

Comment:
We should not use Application to keep our objects. Application class is not designed for that. Instance of it can be recycled at any moment